lib/matplotlib/__init__.py

```diff
@@ -1181,6 +1182,9 @@ def rc_params_from_file(fname, fail_on_error=False, use_default_template=True):
 if rcParams['axes.formatter.use_locale']:
     locale.setlocale(locale.LC_ALL, '')
 
+_log.debug('rcParams: ')
+_log.debug(rcParams)
```

Comment:
Suggested consolidation:
```
_log.debug('rcParams:\n%s', rcParams)
```
I think this will result in a more nicely-formatted output because it will not have the extra header info from the logging formatter at the start of the `rcParams` list.